File: client/src/types/JsonModels/Analytics/FrequentExerciseSummaryModel.ts
export interface FrequentExerciseSummaryModel
{
	exerciseId: number;
	exerciseName: string;
	primaryMuscleGroupId: number;
	primaryMuscleGroupName: string;
	workoutCount: number;
	setCount: number;
	lastTrainedOn: string;
}

File: server/FitMate.Core/JsonModels/Analytics/FrequentExerciseSummaryModel.cs
namespace FitMate.Core.JsonModels.Analytics;

public class FrequentExerciseSummaryModel
{
    public long ExerciseId { get; set; }
    public string ExerciseName { get; set; } = string.Empty;
    public long PrimaryMuscleGroupId { get; set; }
    public string PrimaryMuscleGroupName { get; set; } = string.Empty;
    public int WorkoutCount { get; set; }
    public int SetCount { get; set; }
    public DateTime LastTrainedOn { get; set; }
}
